trillian/integration/ct_integration.go

```diff
@@ -848,7 +847,19 @@ func buildNewPrecertData(cert, issuer *x509.Certificate, signer crypto.Signer) (
 
 // MakeSigner creates a signer using the private key in the test directory.
 func MakeSigner(testDir string) (crypto.Signer, error) {
-	key, err := keyspem.ReadPrivateKeyFile(filepath.Join(testDir, "int-ca.privkey.pem"), "babelfish")
```

Comment:
Was this the only place in the codebase using `keyspem` package? Could add a helper otherwise.

Reply:
There's other uses of this package, but for some different functions, to be addressed in separate PRs. I might add a helper for those, as there's a few repeats...

Reply:
There's other uses of this package, but for some different functions, to be addressed in separate PRs. I might add a helper for those, as there's a few repeats...